content/pop-recovery.md

```diff
@@ -63,7 +63,7 @@ If the existing install is encrypted, please see the [encrypted disk](#encrypted
 
 If the existing OS install needs to be repaired, the installer application should be closed. An app menu is located in the top-left of the screen with the name of the currently running application (in this case: "Install Pop!\_OS"). Click on the app menu and select `Quit`. Alternatively, you can use the installer app to select keyboard and language settings, and then click the `Try Demo Mode` button in the lower-left corner of the install page.
 
->**Note:** Do not choose any install or repair options, as this could result in data loss.
+>**Note:** Do not choose any install or repair options, as this could result in data loss. Also, once you have setup or accessed any of the filesystems, you will need to boot to the recovery environment anew to use any of the install options.
```

Comment:
I feel like these should be two different notes. Tacking that onto a note that warns people _not_ to do an install seems like it just doesn't fit.